index.html

```diff
@@ -36,6 +36,7 @@
             <li><a href="https://t.me/joinchat/AAAAAEAc48wCHkUXViXBcg" target="_blank" rel=external >Telegram</a></li>
             <li><a href="https://www.reddit.com/r/ILUGD/" target="_blank" rel=external >Reddit</a></li>
             <li><a href="https://riot.im/app/#/room/#ilugd:matrix.org" target="_blank" rel=external >Matrix</a></li>
+			<li><a href="https://webchat.freenode.net/?channels=linuxindia" target="_blank" rel=external>IRC on Freenode</a></li>
```

Comment:
Seems fine. 
Just a suggestion. Also try to include the `alt` in the href implementation. It helps the visually impaired get an overview of the link or image. Might not seem important, but there are many visually impaired computer geeks out there. I know a few of them as well.

Reply:
Oh! Sorry, I didn't know it existed.

Kinda awesome it exists :)